docs/commandlineclient.qmd

```diff
@@ -1,51 +1,47 @@
 # iBridges CLI
-The iBridges CLI is a commandline interface to up and download large amounts of data. It only updates data when necessary, i.e. if checksums of files differ.
+The iBridges CLI is a command line interface to up and download large amounts of data. It only updates data when necessary, i.e. if checksums of files differ.
 
 ## Configuration
 
-- Minimal configuration file for uploading data to iRODs:
+The client uses the same configuration files as the GUI, i.e. the *irods_environment.json* and the *ibridges_config.json*.
 
-```ini
-[iRODS]
-irodsenv = /<path to>/irods_environment.json
-irodscoll =
-irodsresc =
-webdav =
-```
-
-- Configuration file for downloading data from iRODS:
-
-```ini
-[iRODS]
-irodsenv = /<path to>/irods_environment.json
-irodscoll =
-irodsresc =
-webdav =
-
-[DOWNLOAD]
-path = /path/to/download/directory
-```
+- Please create a directory named `.irods` in your home directory:
```

Comment:
Just a suggestion.  This could be generically referred to as `~/.irods`.  In Python, even under Windows, the `~/` is recognized as the user's home directory.  It might even be recognized in a **PowerShell**.

Reply:
Oh that is good to know. I will adjust that!

Reply:
I just checked under Windows and, indeed, ~/.irods and ~/.ibridges can be used in **PowerShell** to access these locations.  Tab-completion is also possible with this (universal?) path separator.

Also, L11 below could be simplified to a more generic form:

```qmd
        - Windows: c:/Users/\<username\>/.irods/irods_environment.json
```